### DISCUSSION TITLE
blog and JOSS

### DISCUSSION THREAD

**Discussion by nikosarcevic** [nikosarcevic/CosmOracle#66] [General]
I talked to Robert this morning and he said we should write a short submission for JOSS and towards data science

links:
JOSS https://joss.theoj.org/papers/10.21105/joss.02004

TDS https://towardsdatascience.com/questions-96667b06af5

thoughts

**Comment by lonbar** [comment]
TDS we can definitely do, and shouldn't take too much time. I am not familiar with JOSS, so I don't know what their publishing standards are.

**Comment by marcobonici** [comment]
I have seen a lot of papers on JOSS. Usually they are quite short, so it should not be that long or diffucult.
